readme.md

```diff
@@ -691,7 +696,70 @@ coordsToPair(pairToCoords([1, 2])) // [1, 2]
 pairToCoords(coordsToPair({x: 1, y: 2})) // {x: 1, y: 2}
 ```
 
+### Catamorphism
 
+A `reduceRight` function. Take a bunch of things, and combine them into another. The morphism is from "bunch of things" to "another".
```

Comment:
@stevemao By just looking at this line, I don't have the context as in how 'generalised' we want to be in this repo, but `Catamorphism` can be for other ADTs and not just list. List certainly provides good intuition, but should we mention `bunch of things` could be just one thing here?